extruct/__init__.py

```diff
@@ -1,16 +1,38 @@
+import logging
 from lxml.html import fromstring
-
 from extruct.jsonld import JsonLdExtractor
 from extruct.rdfa import RDFaExtractor
 from extruct.w3cmicrodata import MicrodataExtractor
+from extruct.opengraph import OpenGraphExtractor
+from extruct.microformat import MicroformatExtractor
 from extruct.xmldom import XmlDomHTMLParser
 
 
-def extract(htmlstring, url='http://www.example.com/', encoding="UTF-8"):
+logger = logging.getLogger(__name__)
+
+
+def extract(htmlstring, url='http://www.example.com/', encoding="UTF-8",
+            syntaxes="all", schema_context='http://schema.org'):
     domparser = XmlDomHTMLParser(encoding=encoding)
     tree = fromstring(htmlstring, parser=domparser)
-    return {name: extractor.extract_items(tree, url=url)
-            for name, extractor in (
-                ('json-ld', JsonLdExtractor()),
```

Comment:
This breaks backwards compatibility in the public API - it was "json-ld" before and now is "jsonld", I think it's better to keep it backwards compatible unless there is a strong reason to change it.